src/overworld.c

```diff
@@ -2908,7 +2908,11 @@ static void ZeroObjectEvent(struct ObjectEvent *objEvent)
     memset(objEvent, 0, sizeof(struct ObjectEvent));
 }
 
-static void SpawnLinkPlayerObjectEvent(u8 linkPlayerId, s16 x, s16 y, u8 a4)
+#define linkGender(obj) obj->singleMovementActive
```

Comment:
Why is something called `linkGender` being used to access a field called `singleMovementActive`?